redbot/cogs/trivia/trivia.py

```diff
@@ -251,22 +254,75 @@ def __init__(self):
     @trivia.command(name="list")
     async def trivia_list(self, ctx: commands.Context):
         """List available trivia categories."""
-        lists = set(p.stem for p in self._all_lists())
+        default_lists = sorted([p.resolve().stem for p in get_core_lists()])
+        personal_lists = sorted([p.resolve().stem for p in cog_data_path(self).glob("*.yaml")])
+
         if await ctx.embed_requested():
             await ctx.send(
                 embed=discord.Embed(
                     title=_("Available trivia lists"),
                     colour=await ctx.embed_colour(),
-                    description=", ".join(sorted(lists)),
+                    description=(
+                        bold(_("Default lists"))
+                        + "\n"
+                        + ", ".join(default_lists)
+                        + "\n\n"
+                        + (
+                            bold(_("Custom lists")) + "\n" + ", ".join(personal_lists)
+                            if personal_lists
+                            else ""
+                        )
+                    ),
                 )
             )
         else:
-            msg = box(bold(_("Available trivia lists")) + "\n\n" + ", ".join(sorted(lists)))
+            msg = box(
+                bold(_("Available trivia lists"))
+                + "\n\n"
+                + "- Default lists"
+                + "\n"
+                + ", ".join(default_lists)
+                + (
+                    "\n\n" + "- Custom lists" + "\n" + ", ".join(personal_lists)
+                    if personal_lists
+                    else ""
+                )
+            )
             if len(msg) > 1000:
                 await ctx.author.send(msg)
             else:
                 await ctx.send(msg)
 
+    @commands.is_owner()
+    @trivia.command(name="upload")
+    async def trivia_upload(self, ctx: commands.Context):
+        """Upload a trivia file."""
+        if not ctx.message.attachments:
+            await ctx.send(_("Supply a file with next message or type anything to cancel."))
+            message = await ctx.bot.wait_for("message", check=MessagePredicate.same_context(ctx))
+            if not message.attachments:
+                await ctx.send(_("You have cancelled the upload process."))
+                return
+            parsedfile = message.attachments[0]
+        else:
+            parsedfile = ctx.message.attachments[0]
+        try:
+            await self._save_trivia_list(ctx=ctx, file=parsedfile)
+        except yaml.error.YAMLError as exc:
+            await ctx.send(_("Invalid list uploaded."))
+            LOG.debug(f"File failed to upload: {exc}")
```

Comment:
I think it might be worth sending the syntax error that happened during parsing of the YAML file (based on how permissions cog does it):
```suggestion
        except yaml.MarkedYAMLError as exc:
            await ctx.send(_("Invalid syntax: ") + str(exc))
```
And logging here is not gonna really be needed then as all the needed info is gonna already be in the sent message.